### PR TITLE
Expose Tangram version number

### PR DESCRIPTION
**Pull request by rfriberg**
I'd like to expose the Tangram version number to help with manual (and eventually automated) testing.

### CODE REVIEW

**Comment by rfriberg** [comment]
@hanbyul-here would you mind taking a look at this when you get a chance?

File: src/js/components/tangram.js [hunk 1]
```diff
@@ -4,15 +4,19 @@
 var L = require('leaflet');
 
 var tangramLayerInstance;
+var tangramVersion = '0.11';
```
Comment (hanbyul-here) [inline src/js/components/tangram.js]: Woudn't it be better to take version number from Tangram object? (so we don't have to manually update the version number, also user can see minor version number too!)

Comment (rfriberg) [inline src/js/components/tangram.js]: Yeah, we still have to update manually at the moment. But I do like the idea of grabbing the full (including patch) number from the object.  I didn't see it readily available, but I'll take another look.

Comment (rfriberg) [inline src/js/components/tangram.js]: Updated! (below)

I'm going to leave this `tangramVersion` var here, though, to make our manual updates a little more obvious/intentional-looking.

**Pull request merged**